filebeat/harvester/log.go

```diff
@@ -64,17 +63,9 @@ func (h *Harvester) Harvest() {
 		ts, text, bytesRead, jsonFields, err := readLine(reader)
 		if err != nil {
 			if err == errFileTruncate {
-				seeker, ok := h.file.(io.Seeker)
-				if !ok {
-					logp.Err("can not seek source")
-					return
-				}
-
 				logp.Info("File was truncated. Begin reading file from offset 0: %s", h.Path)
```

Comment:
@urso after our conversation yesterday about warn messages, I will leave it as info, ok?